src/main/java/at/hannibal2/skyhanni/utils/StringUtils.kt

```diff
@@ -120,6 +120,13 @@ object StringUtils {
         return null
     }
 
+    fun List<String>.indexOfFirstMatch(pattern: Pattern): Int? {
```

Comment:
should be in regex utils

Reply:
What even is this, I never wrote that

Reply:
It is in regexUtils. But you didn't merge the hotmAPI again. So you have two now.